polaris-react/src/components/Button/Button.module.scss

```diff
@@ -43,35 +43,55 @@
   user-select: none;
   -webkit-tap-highlight-color: transparent;
 
-  svg {
-    fill: var(--pc-button-icon-fill);
-  }
-
   @media (--p-breakpoints-md-up) {
     font-size: var(--p-font-size-300);
     line-height: var(--p-font-line-height-400);
   }
 }
 
+/* stylelint-disable selector-max-specificity -- Bump specificity to override Icon svg fill on both webpack and vite */
+.Button.Button svg {
```

Comment:
lol. I have never done this before but it makes sense. Is this meant to be temporary and fixed up later? If it is maybe we could add a comment.

Reply:
haha yeah its from [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#increasing_specificity_by_duplicating_selector)! yeah I was on the fence about it being temporary, since it is safer for this to have a greater specificity than `Icon` so the results are consistent no matter what bundler is used. I'm not sure if its possible but I would love to refactor this to either not use `Icon` (so we don't have to override it) or to pass the correct fill into `Icon` directly. I'll add in the temp comment though because technically we could revert this once web prod is on `vite`

Reply:
@alex-page can `Button` be refactored to use `currentColor` and `inherit` ? https://github.com/Shopify/polaris/pull/11273